recipe/meta.yaml

```diff
@@ -26,11 +26,11 @@ requirements:
     - {{ stdlib("c") }}
   host:
     - python
-    - cython <3
+    - cython
     - pip
+    - setuptools
   run:
     - python
-    - pytz
```

Comment:
this was dropped upstream [here](https://github.com/fastavro/fastavro/pull/449/files#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7L100) before the `1.0.0` release was published.